crates/hir-analysis/src/name_resolution/import_resolver.rs

```diff
@@ -530,6 +531,8 @@ impl<'db> ImportResolver<'db> {
     }
 
     fn register_error(&mut self, i_use: &IntermediateUse, err: NameResolutionError) {
+        self.suspicious_imports.remove(&i_use.use_);
```

Comment:
Remove the `use` from the suspicious import set when it turns out to cause another error other than the import-specific ambiguous error that is described in `7. Import Specific Ambiguity`.

Reply:
^ The main change in [Remove duplicated error messages by cleaning suspicious import set](https://github.com/ethereum/fe/pull/873/commits/f1da8c76effd4e43043607bd6d026768519a69ef)